docs/source/user/stencil.rst

```diff
@@ -171,10 +171,10 @@ specified neighborhood, **the behavior is undefined.**
 
 .. _stencil-mode:
 
-``mode``
+``func_or_mode``
 --------
 
-The optional mode parameter controls how the border of the output array
+The optional func_or_mode parameter controls how the border of the output array
```

Comment:
`func_or_mode` should probably have double back ticks around as it looks strange when rendered as it's clearly a parameter to a function (evident from underscores in name).

Reply:
Done! (Sorry for the delay)